source/renderer/app/components/wallet/transactions/WalletTransactionsList.js

```diff
@@ -173,11 +172,15 @@ export default class WalletTransactionsList extends Component<Props> {
   getExpandedTransactions = (): Array<any> =>
     Object.values(this.expandedTransactions);
 
-  renderGroup = (data: TransactionsGroup): Node => (
+  // renderGroup = (data: TransactionsGroup): Node => (
+  // @TX TODO
```

Comment:
@daniloprates what's with this "TODO"?

Reply:
The flow notation for fn return breaks my editor syntax highlighting 😕  fixed it
![image](https://user-images.githubusercontent.com/1504716/98988930-24f58100-2507-11eb-8d1d-f1eb5e75eff2.png)